error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/ReactorTemplates.java

```diff
@@ -271,6 +271,32 @@ Flux<S> after(Flux<T> flux) {
     }
   }
 
+  /** Prefer {@link Flux#onErrorComplete()} over more contrived alternatives */
+  static final class FluxOnErrorComplete<T> {
+    @BeforeTemplate
+    Flux<T> before(Flux<T> flux) {
+      return flux.onErrorResume(e -> Flux.empty());
+    }
+
+    @AfterTemplate
+    Flux<T> after(Flux<T> flux) {
+      return flux.onErrorComplete();
+    }
+  }
+
+  /** Prefer {@link Mono#onErrorComplete()} over more contrived alternatives */
```

Comment:
I was doubting to make the description more specific. However, it's now the same as the Javadoc of surrounding templates so decided to leave it as is.